src/store/modules/collection.ts

```diff
@@ -17,7 +19,20 @@ class Collection extends VuexModule {
   public page = 0;
 
   get getPosts() {
-    return this.items?.filter($ => $.type in PostTypes);
+    if (this.items) {
```

Comment:
I'm always a fan of early returns:

```
if (!this.items) {
    return;
}
```

imo it's easier to read than long if else blocks :)